blivet/formats/fs.py

```diff
@@ -72,6 +72,11 @@ class FS(DeviceFormat):
     _sizeinfoClass = fssize.UnimplementedFSSize
     _syncClass = fssync.UnimplementedFSSync
     _writelabelClass = fswritelabel.UnimplementedFSWriteLabel
+    # This constant is aquired by testing some filesystems
+    # and it's giving us percentage of space left after the format.
+    # This number is more guess then precise number because this
+    # value is already unpredictable and can change in the future...
+    _MagicMetadataMultiplierClass = 1.0
```

Comment:
Yeah and I think ``MetadataSizeFactor`` or something like that would be a better name.